src/pages/sponsor/[uid].astro

```diff
@@ -21,22 +22,157 @@ const descriptionHTML = prismicH.asHTML(sponsor.data.description);
 ---
 
 <Layout
-  title={`${sponsor.data.name} - Guest Speaker - Holiday Snowtacular`}
+  title={`${sponsor.data.name} - Sponsor - Holiday Snowtacular`}
   description={`${sponsor.data.name} is sponsoring the 2022 Holiday Snowtacular in an effort to raise $20,000 for Doctors Without Borders!`}
 >
-  <main>
-    <h1>{sponsor.data.name}</h1>
-    <img src={sponsor.data.logo.url} alt="" />
-    <Fragment set:html="descriptionHTML" />
+  <main class="holiday-supporter">
+    <div class="holiday-supporter-heading">
+      <span class="holiday-supporter-eyebrow">Sponsor</span>
+      <h1 class="holiday-supporter-name">{sponsor.data.name}</h1>
+    </div>
+    <div class="holiday-supporter-logo-container">
+      <img src={sponsor.data.logo.url} alt={`${sponsor.data.name} logo`} />
+    </div>
+    <div class="holiday-supporter-description">
+      <Fragment set:html="descriptionHTML" />
+    </div>
 
     {
-      prismicH.isFilled.link(sponsor.data.link) && (
-        <p>
-          <a href={sponsor.data.link.url}>Visit Site</a>
-        </p>
+      prismicH.isFilled.link(sponsor.data.ctalink) && (
+        <div class="holiday-supporter-cta">
+          <a class="button" data-variant="red" href={sponsor.data.ctalink.url}>
+            {sponsor.data.ctalabel}
+          </a>
+        </div>
       )
     }
+
+    {
+      prismicH.isFilled.group(sponsor.data.links) && (
+        <div class="holiday-supporter-links">
+          <h2>Links</h2>
+          <ul>
+            {sponsor.data.links.map(({ label, link }) => (
+              <>
+                {prismicH.isFilled.link(link) && (
+                  <li>
+                    <a href={link.url} class="holiday-supporter-link">
```

Comment:
This should probably have `rel="sponsored"` on it for Google's sake

Reply:
Oh great call